### DISCUSSION TITLE
Setup Patreon or some other form of croudfunding to have more beers during development?

### DISCUSSION THREAD

**Discussion by Frits1980** [thorrak/fermentrack#692] [Ideas]
Just like the title implies.

**Comment by thorrak** [comment]
Hah! There's always room for more beer in the fridge, but I never want anyone to feel like they need to help me fill it. I enjoy making tools that help people make (hopefully!) better beer. 

There used to be a PayPal link in GitHub, but I think they deprecated it in favor of GitHub Sponsors. I'll take a look at getting that set up if anyone is interested in using it.

**Comment by Frits1980** [comment]
I agree, but I would like to send you my favourite beer to thank you. But to send it all the way from The Netherlands would be a little expensive. But you are free to buy you own favourite of course. Or even better buy ingredients to brew your favourite beer 😀
